core/src/main.cpp

```diff
@@ -264,6 +267,7 @@ constexpr auto default_jsonschema() -> std::string_view
                                 "provider_url",
                                 "mode",
                                 "client_id",
+                                "require_aud_member_from_introspection_endpoint",
                                 "redirect_uri",
```

Comment:
redirect before require in alpha order

Reply:
not terribly important, resolving.

Reply:
Can open an issue for sorting these as it does help with reading/finding things?

Reply:
perhaps - alpha is a good order by default... but there can be strong reasons to clump things (of course, that could be a signal that they should have better names to begin with)...

consistency is good - so yes, please look at these broadly and decide if we should stick to alpha order in the schema and the template.